charts/erigon/values.yaml

```diff
@@ -27,10 +27,70 @@ extraArgsRPCDaemon: []
   #- --http.corsdomain=yourdomain.tld
   #- --http.api=eth,erigon,web3,net,debug,trace,txpool,db
 
-# -- Command replacement for the erigon container
-customCommand: [] # Only change this if you need to change the default command
-
-# -- Command replacement for the rpcdaemon container
+# -- Template used for the default command
+# @default -- See `values.yaml`
+defaultCommandTemplate: |
+  - sh
+  - -ac
+  - >
+  {{- if .Values.p2pNodePort.enabled }}
+    . /env/init-nodeport;
+  {{- end }}
+    exec erigon
+    --datadir=/data
+  {{- if .Values.p2pNodePort.enabled }}
+    {{- if not (contains "--nat=" (.Values.extraArgs | join ",")) }}
```

Comment:
I would remove the = from here. As this way you assume someone would use --nat=extip:<IPaddr>  but erigon can also take --nat extip:<IPaddr> and this wouldn't catch it.

Reply:
That's true. But there would also be the risk that we accidentally catch other flags when removing the `=` . Let's keep this like this for now. In any way, users can now overwrite the default template if they're happy with it.